tck/methods/account.ts

```diff
@@ -0,0 +1,83 @@
+import { AccountCreateTransaction, Hbar, AccountId } from "@hashgraph/sdk";
+
+import { sdk } from "../sdk_data";
+import { getKeyFromString } from "../utils/key";
+import { CreateAccountResponse } from "../response/account";
+
+import { CreateAccountParams } from "../params/account";
+import { applyCommonTransactionParams } from "../params/common-tx-params";
+
+export const createAccount = async ({
+    key,
+    initialBalance,
+    receiverSignatureRequired,
+    maxAutoTokenAssociations,
+    commonTransactionParams,
+    stakedAccountId,
+    stakedNodeId,
+    declineStakingReward,
+    memo,
+    autoRenewPeriod,
+    alias,
+}: CreateAccountParams): Promise<CreateAccountResponse> => {
+    let transaction = new AccountCreateTransaction().setGrpcDeadline(30000);
+
+    if (key != null) {
+        transaction.setKey(getKeyFromString(key));
+    }
+
+    if (initialBalance != null) {
+        transaction.setInitialBalance(Hbar.fromTinybars(initialBalance));
+    }
+
+    if (receiverSignatureRequired != null) {
+        transaction.setReceiverSignatureRequired(receiverSignatureRequired);
+    }
+
+    if (maxAutoTokenAssociations != null) {
+        transaction.setMaxAutomaticTokenAssociations(maxAutoTokenAssociations);
+    }
+
+    if (stakedAccountId != null) {
+        const accountId = AccountId.fromString(stakedAccountId);
+
+        transaction.setStakedAccountId(accountId);
+    }
+
+    if (stakedNodeId != null) {
+        transaction.setStakedNodeId(stakedNodeId);
+    }

```

Comment:
Again Github doesn't let me quote the whole thing but the end of this is row 73. 

Anyway, this might be one of the few really rare cases where passing all these properties in the `AccountCreateTransaction` constructor is a proper use case. The SDK already checks if these are null so you wouldn't have to write all these checks. 

I think this pattern might be useful for you in the SDK so this might have given some life to this SDK feature so we can maintain it.

Reply:
I see what you mean, but some of these parameters still need to be checked for null because we are doing some type conversions in here, for example `Hbar.fromTinybars(initialBalance)` would not work if initialBalance is null. 

Anyways, if it's doable i'm ok with it.

Reply:
It is indeed possible to remove some of these checks. However, I believe it is beneficial to retain them for all parameters.

Reply:
Resolved in private conversation.